unic/utils/src/traits.rs

```diff
@@ -0,0 +1,21 @@
+//! # Unic - Utils - Traits
+//!
+//! A component of [`unic`: Unicode and Internationalization Crates for Rust](/unic/).
```

Comment:
All utils is now one component. We can drop the crate-like doc and only document it as a module, which is what it is now.